tools/platform/gui.cpp

```diff
@@ -273,7 +273,7 @@ void GUI::endFrame(ITransientResourceHeap* transientHeap, IFramebuffer* framebuf
     auto renderEncoder = cmdBuf->encodeRenderCommands(renderPass, framebuffer);
     renderEncoder->setViewportAndScissor(viewport);
 
-    renderEncoder->setPipelineState(pipelineState);
+    renderEncoder->bindPipeline(pipelineState);
```

Comment:
Out of curiosity, why the naming change here (from `set` to `bind` and from `PipelineState` to `Pipeline`)? We probably need to figure out a policy for what our terminology is going to be in `gfx` and try to be consistent about it. Does `bind` mean something different from `set`? If so, what?

Reply:
Originally I was thinking about a name like `beginPipelineBinding`, since the method is really a start of parameter binding. `setPipelineState` sounds more like an one shot, all on its own operation and doesn't feel like what is really going on semantically. I end up with `bindPipeline` which means a little more than just set pipeline state. But can decide to have a more accurate name here.